src/ParcelRegistry/Legacy/Exceptions/ParcelRemovedException.cs

```diff
@@ -1,10 +1,11 @@
-namespace ParcelRegistry
+namespace ParcelRegistry.Legacy.Exceptions
 {
     using System;
     using System.Runtime.Serialization;
+    using Be.Vlaanderen.Basisregisters.AggregateSource;
 
     [Serializable]
-    public sealed class ParcelRemovedException : ParcelRegistryException
```

Comment:
Why remove this class? We still have it in other registries.

Reply:
We didn't want namespace conflicts with the new domain code, and it wasn't necessary given that there is only one exception in legacy.